imagecluster/postproc.py

```diff
@@ -25,11 +25,14 @@ def plot_clusters(clusters, images, max_csize=None, mem_limit=1024**3):
         have (i) enough memory, (ii) many clusters and/or (iii) large
         max(csize) and (iv) max_csize is large or None
     """
+    if len(clusters) < 1:
+        print('No cluster meets the requirements of sim and min_csize, can\'t visualize it.')
+        return
     stats = ic.cluster_stats(clusters)
     if max_csize is not None:
         stats = stats[stats[:,0] <= max_csize, :]
     # number of clusters
-    ncols = stats[:,1].sum()
+    ncols = stats[:, 1].sum()
```

Comment:
Even though not enforced by a spefic code style, this repo used `[i,j]` array indexing w/o white space.